### PR TITLE
Make broker connection overridable through option on command

### PR DESCRIPTION
**Pull request by brnbp**
this pr intends to solve the issue #38 (Choose broker when running consumer)

### CODE REVIEW

File: docs/advanced.md [hunk 1]
```diff
@@ -241,22 +242,34 @@ redirect_stderr=true
...
+
+     And if you need to start the consumption of a topic in a specific offset (it can be useful for debug purposes)
+ you can pass the `--offset=` option, but for this, it will be required to specify the partition too.
+
+    `$ php artisan kafka:consume price-update --partition=2 --offset=34`
+
+
+ - `--broker=`
+
+   Sometimes, you may want to change which broker the consumer should connect into (maybe for testing/debug purposes).
```
Comment (vitorbari) [inline docs/advanced.md]: "should connect to" ?

File: docs/advanced.md [hunk 2]
```diff
@@ -241,22 +242,34 @@ redirect_stderr=true
...
+   For that, you just nedd to call the `--broker` option with another broker connection key already setted in the `config/kafka.php` file.
```
Comment (vitorbari) [inline docs/advanced.md]: *set

File: docs/advanced.md [hunk 3]
```diff
@@ -241,22 +242,34 @@ redirect_stderr=true
 stdout_logfile=/var/log/default/kafka-consumer-price-update.log
 ```
 
-Although you can run this simple command, we've some options you can pass to make it more flexible to your needs.
+##### Options
 
-If you wish do specify in which partition the consumer must be attached, you can set the option `--partition=`
+ Although you can run this simple command, we've some options you can pass to make it more flexible to your needs.
```
[the remark this answers is not in the document]
Comment (brnbp) [reply docs/advanced.md]: fair enough

File: docs/advanced.md [hunk 4]
```diff
@@ -241,22 +242,33 @@ redirect_stderr=true
 stdout_logfile=/var/log/default/kafka-consumer-price-update.log
 ```
 
-Although you can run this simple command, we've some options you can pass to make it more flexible to your needs.
+##### Options
 
-If you wish do specify in which partition the consumer must be attached, you can set the option `--partition=`
+Although you can run this simple command, it has some option you can pass to make it more flexible to your needs.
```
Comment (vitorbari) [inline docs/advanced.md]:
```suggestion
Although you can run this simple command, it provides some options you can pass to make it more flexible to your needs.
```

File: src/Config/Consumer.php [hunk 5]
```diff
@@ -38,9 +38,15 @@ public function __construct(
         string $topic,
         string $consumerGroupId = null,
         int $partition = null,
-        int $offset = null
+        int $offset = null,
+        string $brokerKey = null
     ) {
         parent::__construct($topic);
```
Comment (vitorbari) [inline src/Config/Consumer.php]: You could pass the `$brokerKey` here and then when calling `setBroker` perhaps `$this->setBroker($brokerKey ?: $topicConfig['broker']);`

**Comment by vitorbari** [comment]
closes #38

**Pull request merged**